src/Fixer/Basic/BracesFixer.php

```diff
@@ -846,6 +846,14 @@ private function ensureWhitespaceAtIndexAndIndentMultilineComment(Tokens $tokens
 
         $nextToken = $tokens[$nextTokenIndex];
         if ($nextToken->isComment()) {
+            $previousToken = $tokens[$nextTokenIndex - 1];
+            // do not indent inline comments used to comment out unused code
+            if (
+                0 === strpos($nextToken->getContent(), '//'.$this->whitespacesConfig->getIndent())
+                && $previousToken->isWhitespace() && 1 === preg_match('/\R$/', $previousToken->getContent())
```

Comment:
> For me, while using PHP Storm, commented code looks exactly like this

This is configurable:
![ec515ddd](https://user-images.githubusercontent.com/5972388/29535960-38d0f020-86bc-11e7-94a1-a9ea86cceca7.png)

> if I get you right, you also check that there is no single-line spaces before //, right ?

Only comments proceeded by EOL pass this condition, so basically this PR affects only non-indented comments. If someone will use your PhpStorm configuration, this will not change anything for him (unless he's using crazy single-space indentation).